### PR TITLE
:lipstick: main screen ui

### PR DESCRIPTION
**Pull request by delphinemillet**
- [ ] design action buttons
* add: https://iconmonstr.com/plus-2-svg/
* clear: https://iconmonstr.com/forbidden-1-svg/
- [ ] design modal
- [ ] design dices

### CODE REVIEW

File: src/app.jsx [hunk 1]
```diff
@@ -31,52 +31,47 @@ const App = () => {
       )}
 
       <div className="actions">
-        <button
-          type="button"
-          onClick={() => setPrintModal(!printModal)}
-        >
-          {messages.add}
-        </button>
+        <div className="actions__secondary">
```
Comment (fabienjuif) [inline src/app.jsx]: Why don't you do "actions primary"?
Then in css `.actions.primary` ?

File: src/app.jsx [hunk 2]
```diff
@@ -31,52 +31,47 @@ const App = () => {
...
-          <Fragment>
-            <span className="label">
-              {messages.total}
-            </span>
-            <span>
-              {dices.reduce((acc, { roll = 0 }) => acc + roll, 0)}
-            </span>
-          </Fragment>
-        )}
+      <div className={`stats substract ${dices.length !== 2 && 'inactive'}`}>
```
Comment (fabienjuif) [inline src/app.jsx]: We can use classnames if you prefer it https://bundlephobia.com/result?p=classnames@2.2.6

File: src/app.jsx (hunk 1 above)
Comment (delphinemillet) [reply src/app.jsx]: because `primary` class would define style only for actions, it couldn't be reused

Comment (fabienjuif) [reply src/app.jsx]: you could do a bg-primary and a primary that set bg-color and color

**Pull request merged**